### PR TITLE
Problem: go-ethereum is not updated

### PR DESCRIPTION
**Pull request by mmsqe**
👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

## Summary by CodeRabbit

- **New Features**
  - Implemented a new search bar for enhanced user navigation.
  - Introduced a fresh visual style for the search interface.

- **Bug Fixes**
  - Adjusted gas requirements in relayer precompile to accurately reflect consumption.
  - Updated Ethereum-related configurations to improve transaction processing.

- **Documentation**
  - Updated CHANGELOG with recent modifications and version updates.

- **Refactor**
  - Streamlined parameter key table usage for the EVM module.
  - Optimized fee verification logic to accommodate new Ethereum configurations.

- **Style**
  - Enhanced UI elements for better user experience.

- **Chores**
  - Updated dependencies to the latest versions for improved performance and security.

- **Tests**
  - Added new tests to ensure the reliability of the search functionality.

Please note that internal code changes, filenames, and specific functions have been omitted for confidentiality.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 ### State Machine Breaking
 
 - [#1232](https://github.com/crypto-org-chain/cronos/pull/1232) Adjust require gas in relayer precompile to be closed with actual consumed.
+- [#1247](https://github.com/crypto-org-chain/cronos/pull/1247) Update ethermint to develop, go-ethereum to `v1.11.0`.
```
Comment (mmsqe) [inline CHANGELOG.md:16]:
```suggestion
- [#1247](https://github.com/crypto-org-chain/cronos/pull/1247) Update ethermint to develop, go-ethereum to `v1.11.2`.
```

File: x/cronos/keeper/precompiles/relayer.go [hunk 2]
```diff
@@ -63,13 +66,19 @@ func (bc *RelayerContract) Address() common.Address {
 	return relayerContractAddress
 }
 
+func (bc *RelayerContract) SetChainConfig(isHomestead, isIstanbul, isShanghai bool) {
```
Comment (yihuang) [inline x/cronos/keeper/precompiles/relayer.go:69]: is it called anywhere?

Comment (mmsqe) [reply x/cronos/keeper/precompiles/relayer.go:69]: it's called when [runPrecompiledContract](https://github.com/crypto-org-chain/go-ethereum/pull/13/files#diff-8403dffcbec1accad5356c19a66853e9c4cf77a1a11a2d9eeb73ed0c2022257fR278)

Comment (yihuang) [reply x/cronos/keeper/precompiles/relayer.go:69]: I see, it seems better to call in ethermint ApplyMessage when init, instead of go-ethereum

Comment (mmsqe) [reply x/cronos/keeper/precompiles/relayer.go]: moved the change to [ethermint](https://github.com/crypto-org-chain/ethermint/pull/380/files#diff-feaf82fa09f2e8d80af866070cc26213ea09ec4c89ae8a122a6bb4153f8450d6R83) instead

**Review by yihuang — APPROVED** [review]

**Pull request merged**